### DISCUSSION TITLE
Help with title Regex

### DISCUSSION THREAD

**Discussion by guiluxdj** [Marekkon5/onetagger#288] [Q&A]
Hi,

Here is an example title: artist - title (Original Mix). 

If I use Beatport, I get this syntax; however, if I use Spotify, I get this syntax: **Artist - title - original mix**. 
There is a hyphen instead of 'original mix' within parentheses. 

How can I solve this issue to always have the same syntax 'artist - title (...)'? 
Should I use regex in 'title cleanup'? 

Thank you for your help.

**Comment by Marekkon5** [comment]
Hello, the title cleanup regex is meant for input data. The issue is that Beatport returns title + version, so 1T can format it properly, however Spotify returns `title - original mix` as a whole title, so 1T can't reliably detect this.

**Reply by guiluxdj** [reply]
Thanks for the response, nice job anyway.
I use a second software in addition to reformat the title tag to put it back between ()
